exercises/say/example.go

```diff
@@ -1,45 +1,53 @@
 package say
 
+import "errors"
+
+const testVersion = 1
+
 var small = []string{"zero", "one", "two", "three", "four", "five", "six",
 	"seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen",
 	"fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"}
 var tens = []string{"ones", "ten", "twenty", "thirty", "forty",
 	"fifty", "sixty", "seventy", "eighty", "ninety"}
-var scale = []string{"thousand", "million", "billion",
-	"trillion", "quadrillion", "quintillion"}
+var scale = []string{"thousand", "million", "billion"}
 
-func Say(n uint64) string {
+func Say(n uint64) (string, error) {
```

Comment:
I think this should be changed to a signed `int64`, so we can validate negative numbers also.

I like the change in the function signature to include an error. As we can't return `-1` when `n` is out of range this is the best way to go. :+1: